1.2.c

```diff
@@ -0,0 +1,37 @@
+#define _CRT_SECURE_NO_WARNINGS
+#include <stdio.h>
+#include <math.h>
+
+int main()
+{
+	printf ("Input a: ");
+	int a;
+	scanf ("%d", &a);
+
+	printf ("Input b: ");
+	int b;
+	scanf ("%d", &b);
+
+	int result = 0;
+	int moduleOfa = abs(a);

```

Comment:
```suggestion
	int absoluteValueOfa = abs(a);
```
Модулем абсолютное значение в русском языке на зывают.